### PR TITLE
feat: add clients to shortcuts #703

### LINKED ISSUE #703: Add clients to shortcuts
We require to add clients in the shortcuts because it makes it easier and much more user friendly

Clients added to the shortcuts

Screenshot (23)

### PR DESCRIPTION
**Pull request by Aaryan-kapur**
## Description
Added clients button to the left sidenav

## Related issues and discussion
#703

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const environment = {
   production: false,
   version: env.npm_package_version + '-dev',
   fineractPlatformTenantId: 'default',  // For connecting to server running elsewhere update the tenant identifier
-  baseApiUrl: 'https://demo.openmf.org',  // For connecting to server running elsewhere update the base API URL
```
Comment (spider0061) [inline src/environments/environment.ts]: don't push environment.ts in your PR.

File: src/app/core/shell/sidenav/sidenav.component.html [hunk 2]
```diff
@@ -91,6 +91,20 @@
...
+          <a matLine>Home</a>
+        </mat-list-item>
+        <mat-list-item [routerLink]="['/clients']">
+          <mat-icon matListIcon>
+            <fa-icon icon="users" size="sm"></fa-icon>
+          </mat-icon>
+          <a matLine>Clients</a>
+        </mat-list-item>
+
+
```
Comment (spider0061) [inline src/app/core/shell/sidenav/sidenav.component.html]: remove blank line

File: README.md [hunk 3]
```diff
@@ -29,8 +29,22 @@ git clone https://github.com/openMF/web-app.git
 ```
 npm install
 ```
+6. To prevent any CORS error
```
Comment (spider0061) [inline README.md]: You have pushed these changes in https://github.com/openMF/web-app/pull/705 also.

Comment (Aaryan-kapur) [reply README.md]: Yes, I just realized other changes got added o these too I fixed that sir!

**Comment by karantakalkar** [comment]
Thanks for the PR but we will be closing as toolbar will be revamped in a single PR once either keyboard shortcuts or notifications are in place. Multiple PRs for same feature are redundant.

**Pull request closed without merging**